src/nodes/Preview.vue

```diff
@@ -28,8 +28,10 @@
 		<PreviewOptions v-if="editor.isEditable"
 			:value.sync="value"
 			@update:value="convertToParagraph" />
-		<NcReferenceList :text="node.attrs.href"
-			:limit="1" />
+		<NcReferenceList v-if="!extension.options.isEmbedded"
+			:text="node.attrs.href"
+			:limit="1"
+			:interactive="!extension.options.isEmbedded" />
```

Comment:
The conditions on the `v-if` and the `interactive` are the same.
If we don't display the element at all there's no point in setting interactive to false.
I think we need to pick what we want here.

Reply:
Yes, we needed https://github.com/nextcloud/server/pull/44012#discussion_r1515875561 first, but let me remove the v-if now that this is in

Reply:
merged